server/test/api/fiscalYear.js

```diff
@@ -0,0 +1,82 @@
+/* jshint expr:true*/
+var chai = require('chai');
+var expect = chai.expect;
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+describe('(Fiscal_Years) The Fiscal Years API', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  var newFiscalYear = {
+    label : 'A new Fiscal Year 2017',
+    start_date : new Date('2017-01-01'),
+    number_of_months : 12,
+    note : 'Fiscal Year for Integration Test'
+  };
+
+  var responseKeys = ['id', 'enterprise_id', 'number_of_months', 'label', 'start_date', 'previous_fiscal_year_id', 'locked', 'note'];
+
+  before(helpers.login(agent));
+
+  it('POST /fiscal_years adds a fiscal year', function () {
```

Comment:
Please make sure all your comments show what is actually happening.  In this test a POST request is sent to `/fiscal`, but the comment says `POST /fiscal_years`, which is not true...